tests/unit/lifecycle/LifecycleTask.spec.js

```diff
@@ -299,29 +299,29 @@ describe('lifecycle task helper methods', () => {
                 {
                     Key: 'obj-1',
                     VersionId:
-                    '834373636323233323831333639393939393952473030312020363035',
+                    '834373636323233323831333639393939393952473030312020363030',
```

Comment:
Changed `VersionId` so the version is clearly based by the last few numbers in the id rather than a number in the middle of the id as well as at the end. Easier to manage and see